scripts/installers/make-exe

```diff
@@ -23,6 +24,7 @@ PYINSTALLER_DIR = os.path.join(EXE_DIR, 'pyinstaller')
 ASSETS_DIR = os.path.join(EXE_DIR, 'assets')
 DEFAULT_OUTPUT_ZIP = 'awscli-exe.zip'
 AC_INDEX_INTERNAL_PATH = os.path.join('awscli', 'data', 'ac.index')
+DISTRIBUTION_SOURCE = 'bundle'
```

Comment:
Instead of `bundle`, let's call this `exe` . Mainly `bundle` is a little confusing with v1 bundled installer.